build_resume.js

```diff
@@ -0,0 +1,36 @@
+const readresume = require('./read-schema')
+
+const buildResume = function (resumeObject) {
+  const parsedData = JSON.parse(resumeObject)
+  const basics = parsedData["basics"];
+  const name = basics["name"];
+  const label = basics["label"];
+  const email = basics["email"];
+  const markup = `
```

Comment:
You really did some interesting work over here. I think perhaps you may need to expand the function a bit more considering that there are more keys than basics in the resume JSON object. But i think your approach is very interesting